### PR TITLE
Add itemframes & more signs

### PR DESCRIPTION
**Pull request by wrynvera**
Continuation of #62 
This adds the following;

- Signs
- Arrow Signs
- Hanging Signs
- Hanging Arrow Signs
- Itemframes
- Hanging Itemframes
- Sign Supports

The changes is mostly a collection of various works, the credit and licenses of which, i have indicated appropriately in the new license.txt
Which should cover existing works as well.

Something I regret not being able to do is having the various signs not being an attached node.
But there's no simply non-hacky way to do it because `wallmounted` conflicts with `facedir` or just rotation in general.
I welcome any suggestions for a non-complicated way to achieve this.

Do note that the rotation code used for rotating the arrow signs depends on code from the screwdriver, mainly `on_rotate`

### CODE REVIEW

**Comment by wrynvera** [comment]
Ready for review.
Since its another small change, will merge once 1 dev approval.

**Review by lumberjackgames — APPROVED** [review]
Don't see any issues here.

**Pull request merged**
